### ISSUE TITLE
Bars not showing on some Geckolib mobs

### ISSUE THREAD

**Issue by skel39eek66** [Provismet/ProviHealth#12]
### Type of Error

Incorrect Behaviour

### Mod List

Naturalist
Graveyard

and other mods, where mob animations made via Geckolib...

### Description

Some mobs, that have been done with Geckolib don't show a health bar. For example: Naturalist & Graveyard.

P. S.: Mine Cells works fine, since is requiring owo-lib, not a Geckolib

https://github.com/Provismet/ProviHealth/assets/36078397/08235723-7721-49da-be79-e0fd97a71636

### Mod Version

1.2.1

### Minecraft Version

1.20.1

### Mod Loader

Fabric

### Steps to Reproduce

```bash
1. Spawn a mob
2. Hit 'em (or target on him)
3. You are wonderful!
```

### Crash Log

```bash
No crash report
But here's a log (if needed): https://gist.github.com/skel39eek66/54cb384b1bfe06b4a2f8bad332afece0
```

**Comment by Provismet** [comment]
I've rearranged some hooks and fixed the issue in 1.20.1.

![2023-12-23_16 00 39](https://github.com/Provismet/ProviHealth/assets/17149901/1e3b126a-a886-4220-9185-f3c74586ed1f)

The fix will be live in version 1.2.2 of the mod, after some further testing and feature implementation.

**Comment by skel39eek66** [comment]
Awesome!
